Directory.Build.props

```diff
@@ -13,19 +13,30 @@ See https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?v
   </PropertyGroup>
```

Comment:
General comment: were you able to find a nice way to list the top-level dependencies in our repository? I know we chatted about it previously and I thought there was a way to do it in Azure DevOps, but if you were able to find a way to do so, it would be great to note that somewhere for the person who does the SDL next on our team.

Reply:
Unfortunately it seems they removed whatever let us do that in the past :( I needed to manually go through using the Nuget package manager and check the release date of each of the packages, and update any that were older than 18 months